src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/domain/CaseValidationResponse.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sscs.bulkscancore.domain;
+
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class CaseValidationResponse {
```

Comment:
Need to look at the duplication, this class is covered in here: 
https://github.com/hmcts/core-case-data-store-client/blob/master/src/main/java/uk/gov/hmcts/reform/ccd/client/model/AboutToStartOrSubmitCallbackResponse.java

you may want to keep this for a specific validation scenario but then this class should be deleted
https://github.com/hmcts/sscs-bulk-scan/blob/d22ab15758c745630befb4377c01143e0d03aae0/src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/domain/CcdCallbackResponse.java

Reply:
`CaseValidationResponse` is for validating case even before we start creating cases so not sure how we can use `AboutToStartOrSubmitCallbackResponse.java`  but `CaseValidationResponse`  and `CcdCallbackResponse` contains common attributes. So I think we can extract common attributes to a single class to avoid duplication.

`CcdCallbackResponse.java` is required to return the response to CCD which contains data, errors and warning attributes. So I am not sure if we delete this how we could return the response.

Reply:
`AboutToStartOrSubmitCallbackResponse` is intended for that purpose though?
i.e. why does `CcdCallbackResponse` need to exist?

Reply:
got your point yes CcdCallbackResponse can be removed good point.